connectors/atlassian/install.js

```diff
@@ -31,30 +31,29 @@ fusebit_allowed_return_to=${ctx.body.configuration.fusebit_allowed_return_to}
             files: {
                 'package.json': {
                     engines: {
-                        node: "10"
+                        node: '10',
                     },
                     dependencies: {
-                        "@fusebit/add-on-sdk": "^0.0.1",
-                        "superagent": "^5.2.2"
-                    }
+                        superagent: '^5.2.2',
+                        '@fusebit/add-on-sdk': '^1.0.5',
```

Comment:
1.0.6? 1.3.1?

Reply:
Won't fix, I'm only working on Slack in this change (with the exception of prettifying the other addons)